<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnMovistar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Movistar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLlamando">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUnefon">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Unefon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTelcel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Telcel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnATT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ATT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLLamarTodos">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Llamar Todos</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
